### ISSUE TITLE
ApproachShoot.cs NRF

### ISSUE THREAD

**Issue by cookie966507** [cookie966507/Rangers#29]
I found a null reference exception once the match ended with time while playing against one AI on line 90.

@AnonymousRandomPerson

**Comment by AnonymousRandomPerson** [comment]
I played a 20 second, infinite kill deathmatch against one AI and got no errors. What file is the NRE in?

**Comment by cookie966507** [comment]
![nrf](https://cloud.githubusercontent.com/assets/8390459/14305413/2170b898-fb8a-11e5-9a77-a86905b72058.PNG)

**Comment by cookie966507** [comment]
Mine was a 3 minute deathmatch with infinite kills. It beat me 7-3 :P

**Comment by AnonymousRandomPerson** [comment]
Yeah, I found that earlier today but haven't PRed yet. I'll put it up once I've figured out how to navigate Sheng's maps.
